src/nbl/ext/ImGui/shaders/common.hlsl

```diff
@@ -1,3 +1,6 @@
+// temporary, maybe we should ask user for it and the layout BUT only for sampler + texture binding IDs + set IDs they belong to and size of the texture array?
+#define NBL_MAX_IMGUI_TEXTURES 69
```

Comment:
pass an external pipeline layout when creating your imgui system (you can have a default if none passed), also specify what binding and set will be used